Header header       # for timestamp
float64 accel_value # m/s^2
